content/admin/Advanced Configuration Options.adoc

```diff
@@ -0,0 +1,170 @@
+
+== Changing the location of the cache folder
+
+The system actively uses data caching, which greatly improves its performance.
+A special folder is used to store cached data (compiled templates, email data, etc.)
+and its size can be quite large. By default, this is the cache folder located in the installed system folder.
+If necessary, its location can be changed, for this purpose in a variable
+`cachepass:[_]dir`, located in the file `config.php`, set the absolute path to the new
+location of the cache folder. At the same time, the link pointing to the new cache
+folder should be placed in the same place of the cache folder.
+This is necessary to provide access to some cached system files.
+The new folder must be writable by the user who is running the web server.
+
+== Permanent display of Advanced Filters in List Forms
+
+In SuiteCRM version 7.8, a more compact was introduced instead of Basic and Advanced Search
+functional link:../../user/introduction/user-interface/search/[of filters].
+If you still need a permanent display of filter fields at the top of the List Form
+certain modules (as it was in earlier versions of the system), add the appropriate
+array by analogy with the following example (for modules *Accounts*, *Users* and *Employees*)
+to file `config.php`:
+
+[source,php]
+'enable_legacy_search' =>
+  array (
+    0 => 'Accounts',
+    1 => 'Users',
+    2 => 'Employees',  ),
+
+either the corresponding lines in the file `config_override.php`, example :
+
+[source,php]
+$sugar_config['enable_legacy_search'][] = 'Accounts';
+$sugar_config['enable_legacy_search'][] = 'Users';
+$sugar_config['enable_legacy_search'][] = 'Employees';
+
+
+== Subpanels: replacing the drop-down menu with buttons
+
+If necessary, you can display the functions of a particular
+sub-panel not in the form of drop-down menu items, but in the form of buttons.
+The figure below shows an example of a standard sub-panel
+*Measures* and a modified sub-panel *history* in the module *Pre-contacts*:
+
+image:measures-subpanel.png[Replacing the drop-down menu with buttons]
+
+To change the subpanel *history* do the following :
+
+ . Create a file `custom/Extension/modules/Leads/Ext/Layoutdefs/myCustomization.php`
+ . Add a line to it:
+
+[source,php]
+$layout_defs['Leads']['subpanel_setup']['history']['flat'] = 1
+
+[start=3]
+ . Execute link:../administration-panel/system/[Fast recovery].
+
+== Quickly edit custom combo box
+
+If you need to edit the combo box created in
+link:../administration-panel/developer-tools/[combo box editor],
+then this can be done both with the help of the editor,
+and direct editing of the php file, which may be relevant when editing large amounts of data.
+For example, for dynamic link boxes created in link:../administration-panel/developer-tools/[this example] containing English-language values, you need to edit the file `/custom/include/language/en_us.lang.php`
```

Comment:
The link to the example goes nowhere... you need to rephrase so that you don't mention any example elsewhere, you are just giving a new example right here :-)